ModelBundle/Repository/ContentRepository.php

```diff
@@ -380,7 +380,7 @@ public function countByContentTypeInLastVersion($contentType = null)
      * 
      * @deprecated will be removed in 1.2.0
      */
-    public function findByAuthor($author, $published = null, $limit = null)
+    public function findByAuthor($author, $published = null, $limit = null, $sort = null)
```

Comment:
you are update the wrong method, this method is now deperacted. the nodeController uses now findByAuthorAndSiteId